demos/demo_keygen.py

```diff
@@ -96,6 +96,8 @@ def progress(arg=None):
 
     if options.newphrase:
         phrase = getattr(options, 'newphrase')
+        # Turn it into bytes.
+        phrase=phrase.encode("utf-8")
```

Comment:
can be simplified to

```python
if options.newphrase:
    phrase = options.newphrase.encode("utf-8")
```

... but instead, it may be better to change `_write_private_key()` to use the `b()` helper from `py3compat`, which encodes as utf-8 only if the argument is not already bytes.